dev/devicelab/lib/framework/framework.dart

```diff
@@ -231,3 +262,7 @@ class TaskResult {
     return json;
   }
 }
+
+class TaskResultCheckProcesses extends TaskResult {
+  TaskResultCheckProcesses() : super.success(null);
```

Comment:
What does passing `null` here do?

Reply:
Doesn't set any `data` in the task result - leaves it empty.  Other callers already use `TaskResult.success(null)` if they aren't doing a benchmark and have no other data to report.